src/main/java/teammates/ui/controller/PageData.java

```diff
@@ -1068,6 +1068,16 @@ public String getResponseCommentGiverNameVisibilityString(FeedbackResponseCommen
         }
     }
     
+    public String getPictureUrl(String pictureKey) {
+        if (pictureKey == null || pictureKey.isEmpty()) {
```

Comment:
I'm pretty sure that'll give you a `NullPointerException` if it's `null` since you're trying to access a method on a `null`.